packages/node/tsconfig.json

```diff
@@ -7,11 +7,9 @@
     "module": "nodenext",
     "outDir": "dist",
     "sourceMap": false,
-    "declaration": true,
+    "declaration": false,
```

Comment:
Similarly here, I think this will cause problems for 

```ts
import type { VercelRequest } from '@vercel/node'`
```

Reply:
I changed `types.ts` to be `types.d.ts`, so it doesn't need to be compiled via `tsc` (there was basically nothing to compile anyways, since it's already all types).